doc/cookbook/sub_requests.rst

```diff
@@ -33,7 +33,7 @@ By calling ``handle``, you can make a sub-request manually. Here's an example::
 
 There's some more things that you need to keep in mind though. In most cases
 you will want to forward some parts of the current master request to the
-sub-request. That includes: Cookies, server information, session.
+sub-request like cookies, server information, or the session.
```

Comment:
`, or the sesion.` Serial comma not matching http://symfony.com/doc/current/contributing/documentation/standards.html ;)